src/lein_tools_deps/env.clj

```diff
@@ -24,12 +24,23 @@
       (or exe (throw (ex-info "Could not find clojure executable" {:tried-paths clojure-executables}))))
     "clojure"))
 
+(defn- windows?
+  []
+  (clojure.string/includes? (System/getProperty "os.name") "Windows"))
+
+(defn- cross-platform-sh
+  [exe & args]
+  (if (windows?)
+    (let [{:keys [out] :as result} (apply shell/sh "cmd" "/c" exe args)]
+      (assoc result :out (clojure.string/replace out "\\" "\\\\")))
```

Comment:
And what is the replace for?

Reply:
This is escaping backslashes found in windows paths. Without it read-string screams bloody murder. Should probably add a comment in the code